src/main/java/frc/robot/Constants.java

```diff
@@ -71,18 +81,62 @@
             public static final int kSpinnerMotorPort = 9; // motor for the wheel that spins the control panal
 
             public static final int kColorSensorPort = 0; // Color sensor that senses the colors on the control panel
+
+            /*values the color sensor values to detect the colors on control panel
+            red*/
+            public static final double kRedTargetR = 0.476;
+            public static final double kRedTargetG = 0.376;
+            public static final double kRedTargetB = 0.15;
+            //Yellow
```

Comment:
Add a space before comments to make it easier to read.
```suggestion
            // Yellow
```